src/styles/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  position: relative;
+  display: flex;
+  gap: 4px;
+  width: 80px;
+  height: 16px;
+  background: url(../images/star.svg) repeat-x 0 0 / 16px;
```

Comment:
Please verify that the path to the image (`../images/star.svg`) is correct and that the image exists in the specified directory. This is crucial for the background image to be displayed properly.